alpha/apps/kaltura/modules/kmc/actions/extloginAction.class.php

```diff
@@ -42,6 +42,8 @@ public function execute()
 		$ksObj = kSessionUtils::crackKs($ks);
 		$ksPartnerId = $ksObj->partner_id;
 		
+		if($ksObj->getPrivilegeByName(kSessionBase::PRIVILEGE_DISABLE_PARTNER_CHANGE_ACCOUNT && $requestedPartnerId != $ksPartnerId))
```

Comment:
should be getPrivilegeByName(kSessionBase::PRIVILEGE_DISABLE_PARTNER_CHANGE_ACCOUNT) ...